src/server/index.js

```diff
@@ -4,6 +4,9 @@ import hapi from '@hapi/hapi'
 import Scooter from '@hapi/scooter'
 import { Engine as CatboxRedis } from '@hapi/catbox-redis'
 
+// eslint-disable-next-line n/no-unpublished-import
+import { Engine as CatboxMemory } from '@hapi/catbox-memory'
```

Comment:
This should be:
- A dev dependency in tests/test-helpers
- Marked as such in eslint file

I'll also have a play with this tomororrw